#!/usr/bin/env python
PACKAGE = "project"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

euler_const = gen.const("Euler", int_t, 0, "euler integration method")
runge_kutta_const = gen.const("Runge_Kutta", int_t, 1, "Runge-Kutta integration method")

integr_enum = gen.enum([euler_const, runge_kutta_const], 
	"an enum which represents the integration method used to calculate odometry")

gen.add("integr_method", int_t, 0, "a parameter to set the integration method used to calculate odometry", 
		0, 0, 1, edit_method = integr_enum)

exit(gen.generate(PACKAGE, "odometry_computer", "parameters"))